tests/testthat/test-TealDatasetConnector.R

```diff
@@ -413,7 +413,7 @@ testthat::test_that("fun_cdisc_dataset_connector", {
     )
     x$w <- as.numeric(rnorm(40, 0, 1))
     x$ww <- as.numeric(rnorm(40, 0, 1))
-    formatters::var_labels(x) <- c("STUDYID", "USUBJID", "z", "zz", "NAs", "w", "ww")
+    teal.data::formatters_var_labels(x) <- c("STUDYID", "USUBJID", "z", "zz", "NAs", "w", "ww")
```

Comment:
```suggestion
    var_labels(x) <- c("STUDYID", "USUBJID", "z", "zz", "NAs", "w", "ww")
```

Reply:
Hey @gogonzo GitHub is having issues today as this was update in a commit, that is not visible in PRs summary of edited files lol 1d7d88ebfd4b4334e1c3ed4fdbb0d8a9f600991f - chaning to col_labels anyway

Reply:
7b7ade64166f19ae0aeb670f1b40c2b6d771a11f